appletree/plugins/efficiency.py

```diff
@@ -64,7 +64,7 @@ def simulate(self, key, parameters, s1_area):
 @takes_config(
     SigmaMap(
         name="s2_cut_acc",
-        default=["_s2_cut_acc.json", "_s2_cut_acc.json", "_s2_cut_acc.json"],
+        default=["_s2_cut_acc.json", "_s2_cut_acc.json", "_s2_cut_acc.json", "s2_cut_acc_sigma"],
```

Comment:
The modification is just to check the functionality of `SigmaMap`, introduced by https://github.com/XENONnT/appletree/pull/116.